### PR TITLE
update the basic feature tables for v6.4

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by ljun0712 — APPROVED** [review]

File: basic-features.md [hunk 1]
```diff
@@ -10,156 +10,157 @@ aliases: ['/docs-cn/dev/basic-features/']
...
+| [快速导入 (TiDB Lightning)](/tidb-lightning/tidb-lightning-overview.md) |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |
+| mydumper 逻辑导入                                                       | 已废弃| 已废弃| 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 |
+| [Dumpling 逻辑导入](/dumpling-overview.md)                              |   Y   |   Y   |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |
+| [事务 `LOAD DATA`](/sql-statements/sql-statement-load-data.md)          |   Y   |   Y   |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    | N [^3] |
+| [数据迁移工具](/migration-overview.md)                                  |   Y   |   Y   |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)                     |   Y   |   Y   |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |
+| [Change data capture (CDC)](/ticdc/ticdc-overview.md)                   |   Y   |   Y   |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |
 
 ## 管理，可视化和工具
```
Comment (IANTHEREAL) [inline basic-features.md:165]: add
```
| [基于 EBS 的备份和恢复](https://docs.pingcap.com/zh/tidb-in-kubernetes/v1.4/volume-snapshot-backup-restore)                        |    Y    |    N     |    N     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
| [PITR](/br//br-pitr-guide)   |    Y    |    Y     |    Y     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
```

**Review by joey-yez — APPROVED** [review]
LGTM

**Review by CbcWestwolf — APPROVED** [review]

File: basic-features.md [hunk 2]
```diff
@@ -10,156 +10,162 @@ aliases: ['/docs-cn/dev/basic-features/']
...
+| [`AUTO_RANDOM` 列](/auto-random.md)                          |    Y   |    Y   |    Y   |    Y    |    Y   |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [DDL 算法断言](/sql-statements/sql-statement-alter-table.md) |    Y   |    Y   |    Y   |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| 在单条语句中添加多列                                         |   Y  |   Y  | Y | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
+| [更改列类型](/sql-statements/sql-statement-modify-column.md) |    Y   |    Y   |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    N     |    N     |
+| [临时表](/temporary-tables.md)                               |    Y   |    Y   |    Y    |    Y    |    Y    |    Y     |    Y     |    N     |    N     |    N     |    N     |
+| [并行 DDL](/system-variables.md#系统变量##变量参考###tidb_enable_concurrent_ddl-从-v620-版本开始引入)  |    Y   |    Y   |    Y    |    N    |    N    |    N     |    N     |    N     |    N     |    N     |    N     |
```
Comment (qiancai) [inline basic-features.md:99]:
```suggestion
| [并行 DDL](/system-variables.md#tidb_enable_concurrent_ddl-从-v620-版本开始引入)  |    Y   |    Y   |    Y    |    N    |    N    |    N     |    N     |    N     |    N     |    N     |    N     |
```

File: basic-features.md [hunk 3]
```diff
@@ -10,156 +10,162 @@ aliases: ['/docs-cn/dev/basic-features/']
...
+| [添加索引加速](/system-variables.md#系统变量##变量参考###tidb_ddl_enable_fast_reorg-从-v630-版本开始引入)  | 实验特性 | 实验特性 |    N    |    N    |    N    |    N     |    N     |    N     |    N     |    N     |    N     |
```
Comment (qiancai) [inline basic-features.md:100]:
```suggestion
| [添加索引加速](/system-variables.md#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入)  | 实验特性 | 实验特性 |    N    |    N    |    N    |    N     |    N     |    N     |    N     |    N     |    N     |
```

File: basic-features.md [hunk 4]
```diff
@@ -189,4 +194,6 @@ aliases: ['/docs-cn/dev/basic-features/']
 
 [^2]: TiDB 支持的完整 SQL 列表，见[语句参考](/sql-statements/sql-statement-select.md)。
 
-[^3]: 对于 TiDB v4.0，事务 `LOAD DATA` 不保证原子性。
+[^3]: 从 TiDB v6.4 开始，支持[高性能单调递增`AUTO_INCREMENT` 列](/auto-increment.md#MySQL兼容模式)
```
Comment (TomShawn) [inline basic-features.md:197]:
```suggestion
[^3]: 从 TiDB v6.4 开始，支持[高性能单调递增 `AUTO_INCREMENT` 列](/auto-increment.md#mysql-兼容模式)
```

**Review by shichun-0415 — COMMENTED** [review]
add EBS br and PITR

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: basic-features.md [hunk 5]
```diff
@@ -10,156 +10,161 @@ aliases: ['/docs-cn/dev/basic-features/']
...
+| [执行计划缓存](/sql-prepared-plan-cache.md)                          |    Y    |    Y    |    Y    |    Y    |    Y    |    Y     |    Y     | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
+| [执行计划管理 (SPM)](/sql-plan-management.md)                       |    Y    |    Y    |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
+| [下推计算结果缓存 (Coprocessor Cache)](/coprocessor-cache.md)       |    Y    |    Y    |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     | 实验特性 |
+| [Stale Read](/stale-read.md)                                        |    Y    |    Y    |    Y   |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    N     | N        |
+| [Follower Read](/follower-read.md)                                  |    Y    |    Y    |    Y   |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
+| [通过系统变量 tidb_snapshot 读取历史数据](/read-historical-data.md) |    Y    |    Y    |    Y   |    Y    |   Y     |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
```
Comment (Oreoxmt) [inline basic-features.md:79]:
```suggestion
| [通过系统变量 `tidb_snapshot` 读取历史数据](/read-historical-data.md) |    Y    |    Y    |    Y   |    Y    |   Y     |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
```

File: basic-features.md [hunk 6]
```diff
@@ -184,9 +189,13 @@ aliases: ['/docs-cn/dev/basic-features/']
...
 | [前台限流](/tikv-configuration-file.md#前台限流)                            | Y | Y | Y | 实验特性 | 实验特性 |    N     |    N     |    N     |    N     |    N     | N        |
+| [基于 EBS 的备份和恢复](https://docs.pingcap.com/zh/tidb-in-kubernetes/v1.4/volume-snapshot-backup-restore)                        |    Y    |    N     |    N     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
+| [PITR](/br/br-pitr-guide.md)   |    Y    |    Y     |    Y     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
 
 [^1]: TiDB 误将 latin1 处理为 utf8 的子集。见 [TiDB #18955](https://github.com/pingcap/tidb/issues/18955)。
 
 [^2]: TiDB 支持的完整 SQL 列表，见[语句参考](/sql-statements/sql-statement-select.md)。
 
-[^3]: 对于 TiDB v4.0，事务 `LOAD DATA` 不保证原子性。
+[^3]: 从 TiDB v6.4 开始，支持[高性能单调递增 `AUTO_INCREMENT` 列](/auto-increment.md#mysql-兼容模式)
```
Comment (Oreoxmt) [inline basic-features.md:199]:
```suggestion
[^3]: 从 TiDB v6.4.0 开始，支持[高性能、全局单调递增的 `AUTO_INCREMENT` 列](/auto-increment.md#mysql-兼容模式)
```

File: basic-features.md [hunk 7]
```diff
@@ -10,156 +10,161 @@ aliases: ['/docs-cn/dev/basic-features/']
...
+| [更改列类型](/sql-statements/sql-statement-modify-column.md) |    Y   |    Y   |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    N     |    N     |
+| [临时表](/temporary-tables.md)                               |    Y   |    Y   |    Y    |    Y    |    Y    |    Y     |    Y     |    N     |    N     |    N     |    N     |
+| [并行 DDL](/system-variables.md#tidb_enable_concurrent_ddl-从-v620-版本开始引入)  |    Y   |    Y   |    Y    |    N    |    N    |    N     |    N     |    N     |    N     |    N     |    N     |
+| [添加索引加速](/system-variables.md#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入)  | 实验特性 | 实验特性 |    N    |    N    |    N    |    N     |    N     |    N     |    N     |    N     |    N     |
+| [元数据锁](/metadata-lock.md)                                  | 实验特性 | 实验特性 |    N    |    N    |    N    |    N     |    N     |    N     |    N     |    N     |    N     |
+| [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-to-timestamp.md)  | 实验特性 |    N    |    N    |    N    |    N    |    N     |    N     |    N     |    N     |    N     |    N     |
```
Comment (Oreoxmt) [inline basic-features.md:102]:
```suggestion
| [`FLASHBACK CLUSTER TO TIMESTAMP`](/sql-statements/sql-statement-flashback-to-timestamp.md)  | 实验特性 |    N    |    N    |    N    |    N    |    N     |    N     |    N     |    N     |    N     |    N     |
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Comment by IANTHEREAL** [comment]
LGTM

File: basic-features.md [hunk 8]
```diff
@@ -10,156 +10,161 @@ aliases: ['/docs-cn/dev/basic-features/']
...
+| --------------------------------------------------------- | :------: | :------: | :------: | :------: | :------: | :------: | :------: | :------: | :------: | :------: | -------- |
+| [CM-Sketch](/statistics.md)                               | 默认关闭 | 默认关闭 | 默认关闭 | 默认关闭 | 默认关闭 |  默认关闭  |  默认关闭  |  Y  |  Y  |  Y  | Y        |
+| [直方图](/statistics.md)                                  |   Y  |   Y  |   Y   |   Y   |   Y   |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
+| 扩展统计信息（多列）                    | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | N        |
+| [统计反馈](/statistics.md#自动更新)                       | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 |  已废弃  | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
+| [统计信息自动更新](/statistics.md#自动更新) |   Y |   Y |   Y  |   Y   |   Y   |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
+| [快速分析](/system-variables.md#tidb_enable_fast_analyze) | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
+| [动态裁剪](/partitioned-table.md#动态裁剪模式) | Y | Y | Y | Y | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | N | N |
 
 ## 安全
```
[the remark this answers is not in the document]
Comment (ljun0712) [reply basic-features.md:141]: SM4只是其中一个加密算法，TiKV在v6.3支持的，TiFlash在v6.4支持的，所以不建议展开一行

**Pull request merged**